cmd/eksctl-anywhere/cmd/deletecluster.go

```diff
@@ -45,23 +41,11 @@ var deleteClusterCmd = &cobra.Command{
 	},
 }
 
-func preRunDeleteCluster(cmd *cobra.Command, args []string) error {
-	cmd.Flags().VisitAll(func(flag *pflag.Flag) {
-		err := viper.BindPFlag(flag.Name, flag)
-		if err != nil {
-			log.Fatalf("Error initializing flags: %v", err)
-		}
-	})
-	return nil
-}
-
 func init() {
 	deleteCmd.AddCommand(deleteClusterCmd)
+	addClusterFlags(deleteClusterCmd, &dc.clusterOptions, "filename", "install-packages")
 	deleteClusterCmd.Flags().StringVarP(&dc.fileName, "filename", "f", "", "Filename that contains EKS-A cluster configuration, required if <cluster-name> is not provided")
```

Comment:
this one is gonna be added twice

Reply:
added filename as an override to addClusterFlags and left it to be initialized by deletecluster.go since the usage text for the delete cmd is slightly different

Reply:
Backed out the flag grouping changes